projects/elife.yaml

```diff
@@ -833,6 +835,12 @@ observer:
                     - bus-articles--{instance}
                     - bus-metrics--{instance}
     aws-alt:
+        masterless:
```

Comment:
would rename the `masterless` key here (which is just the environment name) to avoid confusion with the `masterless: true` configuration, e.g. `masterless-sample`